agent/config/config.go

```diff
@@ -0,0 +1,29 @@
+package config
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/viper"
+)
+
+type Config struct {
+	ServerURL string `json:"serverURL"`
+	DevMode   bool   `json:"devMode"`
+	APIKey    string `json:"apiKey"`
+}
+
+func LoadConfig() (Config, error) {
+	viper.SetEnvPrefix("tracetest")
+	viper.SetEnvKeyReplacer(strings.NewReplacer(".", "_"))
+	viper.AutomaticEnv()
```

Comment:
The idea is to have the agent to be loaded via flags `--apiKey` or via env variables `TRACETEST_APIKEY`

Reply:
As I'm not sure how we gonna handle configuration with files yet, I'll add support for that later